tests/data/contained_object_schema.yaml

```diff
@@ -1,5 +1,5 @@
 $schema: http://json-schema.org/draft-07/schema#
-$id: https://github.com/lsst-ts/ts_scheduler/blob/master/schema/Scheduler.yaml
+$id: https://github.com/lsst-ts/ts_scheduler/blob/main/schema/Scheduler.yaml
```

Comment:
You found an existing error. Please change this to:
```
$id: https://github.com/lsst-ts/ts_salobj/blob/master/tests/data/contained_object_schema.yaml
```

Reply:
fixed